depmapomics/config_prod.py

```diff
@@ -63,19 +63,27 @@
 TAIGA_FP_FILENAME = "fingerprint_lod_matrix"
 
 
-########### Gumbo configs #############
-GUMBO_SHEET = "https://docs.google.com/spreadsheets/d/1esHqP0Z76_oVMhQtaigyF9FuuniAQefkANoUzLD9rt4"
-GUMBO_SHEETNAME = "Gumbo"
-
 # upload mapping, taiga latest to file name dicts
-LATEST2FN_NUMMAT = {
-    TAIGA_CN: VIRTUAL_FILENAMES_NUMMAT_CN,
-    TAIGA_EXPRESSION: VIRTUAL_FILENAMES_NUMMAT_EXP,
-    TAIGA_MUTATION: VIRTUAL_FILENAMES_NUMMAT_MUT,
+LATEST2FN_NUMMAT_MODEL = {
```

Comment:
isn't it on global?

Reply:
No because the TAIGA_CN and TAIGA_MUTATION stuff is different between dev and master